build.ps1

```diff
@@ -3,6 +3,7 @@ $ErrorActionPreference = 'Stop'
 
 # See https://github.com/lukesampson/scoop/issues/3859
 Set-Strictmode -Off
+.\dev-env\windows\bin\dadew.ps1 uninstall
```

Comment:
this doesn’t seem entirely reliable in fixing a previously broken node since sometimes the msys uninstall fails but seems at least worth keeping until we rotated all nodes.